Documentation/Blazorise.Docs/Components/DocsPageApi.razor

```diff
@@ -0,0 +1,20 @@
+﻿<DocsPageSubtitle>
+    API
+</DocsPageSubtitle>
+
+<DocsPageParagraph>
+    See the documentation below for a complete reference to all of the props and classes available to the components mentioned here.
+</DocsPageParagraph>
+
+<DocsPageParagraph>
+    <UnorderedList>
+        <UnorderedListItem>
+            <Link To="@Link">@Name</Link>
+        </UnorderedListItem>
+    </UnorderedList>
+</DocsPageParagraph>
+@code {
+    [Parameter] public string Name { get; set; }
```

Comment:
Instead of having a single parameters it would be better to introduce additional items components, named `DocsPageApiItem.`

The use case would be:

```razor
<DocsPageApi>
  <DocsPageApiItem Url="docs/some-component-page" Name="<ComponentName />">
  <DocsPageApiItem Url="docs/some-component-page2" Name="<ComponentName2 />">
  <DocsPageApiItem Url="docs/some-component-page3" Name="<ComponentName3 />">
</DocsPageApi>
```